templates/studygroups/email/weekly-update.html

```diff
@@ -374,5 +379,9 @@ <h2>
     </div>
   </div>
 
+{% block scripts %}
+    <script src="{% static 'js/vendor/pygal-tooltips.min.js' %}"></script>
```

Comment:
this will only work for the online version I assume?

Reply:
yes that's right